src/components/buttons/ButtonsContainer.tsx

```diff
@@ -4,6 +4,8 @@ import { View } from 'react-native'
 import { styled } from '../hoc/styled'
 import { MainButton, MainButtonType } from '../themed/MainButton'
 
+const BUTTON_MARGINS = [0.5, 0, 0.5, 0]
```

Comment:
This works like the web, so I think `[0.5, 0]` would do the same thing. On the web, a 2-item array means vertical / horizontal, as opposed to giving each side separately.